pallets/teerex/src/lib.rs

```diff
@@ -344,7 +344,7 @@ pub mod pallet {
 		pub fn shield_funds(
 			origin: OriginFor<T>,
 			incognito_account_encrypted: Vec<u8>,
-			amount: BalanceOf<T>,
+			#[pallet::compact] amount: BalanceOf<T>,
```

Comment:
I think we can leave out `shield_funds` and `unshield_funds` as these two are not called by us :)